gulpfile.babel.js

```diff
@@ -267,7 +267,6 @@ gulp.task('metadata', () => {
 gulp.task('default', ['clean'], cb => {
   runSequence(
     ['styles', 'styles-grid'],
-    ['scripts'],
```

Comment:
With that in mind, should we just pull this in now since it does function as expected and keeps builds minimal. Then in the rewrite we can set things up The Proper Way :tm: